### PR TITLE
Backport 4.3: Orphaned packages #25067 keichwa (#3329) (#3338)

### PR DESCRIPTION
**Pull request by keichwa**
* orphaned packages in Client Configuration Guide
* https://github.com/SUSE/spacewalk/issues/25067
* https://bugzilla.suse.com/show_bug.cgi?id=1227882
Co-authored-by: Marek Czernek <marek.czernek@suse.com>
* Update package-management.adoc
* add link from reference to client config guide

# Some hints

Consider adding an entry to the `CHANGELOG.md` file in the toplevel directory.
Cosmetic changes such as fixing typos do not need log entries (nevertheless it is important to fix typos, etc.)!
In the `manager-4.3`, the hidden `.changelog` file with a leading dot is still in use.

Add Description, Target branches, and related Links below the following section titles.

In the Description, just enter a summary of why you created this PR (and, if available, add any relevant diagram).

# Description

Short summary of why you created this PR (if you added documentation, please add any relevant diagram).

# Target branches

* Which product version this PR applies to (Uyuni, SUMA 4.3, SUMA MU X.Y.Z, or SUMA development version).  This information can be helpful if `ifeval` statements are needed to publish it for certain products only.
* Does this PR need to be backported? If yes, create an issue for tracking it and add the link to this PR.
* Whenever possible, cross-reference each backport PR here, so that all backports can be easily accessed from the description.

Backport targets (edit as needed):

- master
- 4.3

# Links
- This PR tracks issue #<insert spacewalk issue, if any>
- Related development PR #<insert PR link, if any>

### CODE REVIEW

File: modules/client-configuration/pages/package-management.adoc [hunk 1]
```diff
@@ -40,3 +40,51 @@ When the profile has been created, you can use it to install more clients with t
...
+This means that you can delete orphaned packages only after the Taskomatic job execution.
+If you encounter an orphaned package that you cannot delete, wait 24 hours and try the deletion again.
+
+// .Procedure: Deleting orphaned packages
+
+You can deleted orphaned packages:
+
+// * With the UI (FIXME is this possible?  How?
+
+* with commandline tools targeting individual packages.
```
Comment (0rnela) [inline modules/client-configuration/pages/package-management.adoc:77]:
```suggestion
* with command line tools targeting individual packages.
```

File: modules/client-configuration/pages/package-management.adoc [hunk 2]
```diff
@@ -40,3 +40,51 @@ When the profile has been created, you can use it to install more clients with t
...
+
+You can view orphaned packages in the {webui} by clicking menu:Software[Manage > Packages > view Packages in no channel]
+
+{productname} periodically runs a Taskomatic job that searches for orphaned packages and modifies the package organization ID to 1.
+This means that you can delete orphaned packages only after the Taskomatic job execution.
+If you encounter an orphaned package that you cannot delete, wait 24 hours and try the deletion again.
+
+// .Procedure: Deleting orphaned packages
+
+You can deleted orphaned packages:
```
Comment (0rnela) [inline modules/client-configuration/pages/package-management.adoc:73]:
```suggestion
Delete orphaned packages:
```

File: modules/client-configuration/pages/package-management.adoc [hunk 3]
```diff
@@ -40,3 +40,51 @@ When the profile has been created, you can use it to install more clients with t
...
+
+* with commandline tools targeting individual packages.
+  For example:
++
+
+----
+spacecmd package_remove zypper-1.14.52-150400.1.9.x86_64
+----
+
+* with commandline tools targeting all orphaned packages at once:
```
Comment (0rnela) [inline modules/client-configuration/pages/package-management.adoc:85]:
```suggestion
* with command line tools targeting all orphaned packages at once:
```

File: modules/reference/pages/software/manage-packages.adoc [hunk 4]
```diff
@@ -6,4 +6,5 @@ The menu:Software[Manage > Packages] section allows you to manage packages that
 Select a channel from the drop-down box to see all packages related to that channel.
 If you have administration privileges within your organization, you can also delete packages.
 
-For more information about custom channels, see xref:administration:custom-channels.adoc[].
\ No newline at end of file
+* For handling orphaned packages that are packages synchronized by {productname}, but not associated with a software channel, see xref:client-configuration:package-management.adoc#package-management-orphaned-packages[].
```
Comment (0rnela) [inline modules/reference/pages/software/manage-packages.adoc:9]:
```suggestion
* For handling orphaned packages that are synchronized by {productname}, but not associated with a software channel, see xref:client-configuration:package-management.adoc#package-management-orphaned-packages[].
```

File: modules/client-configuration/pages/package-management.adoc [hunk 5]
```diff
@@ -40,3 +40,51 @@ When the profile has been created, you can use it to install more clients with t
...
+
+Orphaned packages are packages synchronized by {productname} that are not associated with a software channel.
+Therefore, orphaned packages are usually not available to {productname} clients and users cannot install such packages without additional effort.
+
+A package can become orphaned as a result of one of the following events:
+
+
+* A synchronized repository removes a package.
+  By default, {productname} 5.0 unlinks such package from the channel, but does not remove the package.
+  In {productname} 4.3, you can enable this behavior by using the [literal]``strict mode`` channel configuration.
```
Comment (0rnela) [inline modules/client-configuration/pages/package-management.adoc:57]: I would NOT keep this in the 5.0 version, it only confuses the user. Rather, have ONLY relevant instructions in the documents that are either 5.0 or 4,3.

File: modules/client-configuration/pages/package-management.adoc [hunk 6]
```diff
@@ -40,3 +40,51 @@ When the profile has been created, you can use it to install more clients with t
...
+* A repository supersedes a package with a newer version and thus removes the previous version.
+* The reposync process got interrupted (for example, because of an out of space exception) and thus downloaded packages were not associated with a channel.
+* Users manually uploaded packages, but did not associate them with any channel.
+
+Orphaned packages cost space in user environment and cannot be easily distributed to clients because they are not associated with a channel.
+Orphaned package can make sense with specific workflows like bootstrapping or customer-specific procedures.
+
+You can view orphaned packages in the {webui} by clicking menu:Software[Manage > Packages > view Packages in no channel]
+
+{productname} periodically runs a Taskomatic job that searches for orphaned packages and modifies the package organization ID to 1.
```
Comment (0rnela) [inline modules/client-configuration/pages/package-management.adoc:67]:
```suggestion
{productname} periodically runs a taskomatic job that searches for orphaned packages and modifies the package organization ID to 1.
```

File: modules/client-configuration/pages/package-management.adoc [hunk 7]
```diff
@@ -40,3 +40,51 @@ When the profile has been created, you can use it to install more clients with t
...
+This means that you can delete orphaned packages only after the Taskomatic job execution.
```
Comment (0rnela) [inline modules/client-configuration/pages/package-management.adoc:68]:
```suggestion
This means that you can delete orphaned packages only after the taskomatic job execution.
```

**Review by 0rnela — CHANGES_REQUESTED** [review]

File: modules/client-configuration/pages/package-management.adoc (hunk 5 above)
Comment (keichwa) [inline modules/client-configuration/pages/package-management.adoc:57]:
```suggestion
  With  [literal]``strict mode`` channel configuration, {productname} unlinks such a package from the channel, but will not remove the package.
```

Comment (keichwa) [reply modules/client-configuration/pages/package-management.adoc]: ok, done!

File: modules/reference/pages/software/manage-packages.adoc [hunk 8]
```diff
@@ -6,4 +6,5 @@ The menu:Software[Manage > Packages] section allows you to manage packages that
 Select a channel from the drop-down box to see all packages related to that channel.
 If you have administration privileges within your organization, you can also delete packages.
 
-For more information about custom channels, see xref:administration:custom-channels.adoc[].
\ No newline at end of file
+* For handling orphaned packages that are synchronized by {productname}, but not associated with a software channel, see xref:client-configuration:package-management.adoc#package-management-orphaned-packages[].
```
Comment (keichwa) [inline modules/reference/pages/software/manage-packages.adoc:9]:
```suggestion
* Orphaned packages are packages that are synchronized by {productname}, but not associated with a software channel.
  For handling those packages, see xref:client-configuration:package-management.adoc#package-management-orphaned-packages[].

```

**Pull request merged**